open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlUI.java

```diff
@@ -0,0 +1,246 @@
+        HBox r3 = controlT.createCustomText("Ctrl");
+        r3.getChildren().addAll(new ImageView(myPlusDir), controlT.createCustomText("Y"));
+
+        VBox l4 = controlT.createLeftLabel("Collect Garbage ");
+        HBox r4 = controlT.createCustomText("g");
+
+        VBox l5 = controlT.createLeftLabel("Cancel Query");
+        HBox r5 = controlT.createCustomText("Esc");
+
+        dispArea.add(l1, 0, 1);
+        dispArea.add(r1, 2, 1);
+        dispArea.add(l2, 0, 2);
+        dispArea.add(r2, 2, 2);
+        dispArea.add(l3, 0, 3);
+        dispArea.add(r3, 2, 3);
+        dispArea.add(l4, 0, 4);
+        dispArea.add(r4, 2, 4);
+        dispArea.add(l5, 0, 5);
+        dispArea.add(r5, 2, 5);
+        return mainBox;
+    }
+
+    /**
+     * Creates a sub-window containing Map Zoom Controls.
+     * 
+     * @return mainBox the sub-window contained in a GridPane.
+     */
+    private GridPane createMapZoom()
+    {
+        ControlTab thecontrols = new ControlTab();
+        GridPane mainBox = thecontrols.createBlankPane(" Zoom Controls");
+        GridPane dispArea = thecontrols.getDisplayArea();
+
+        VBox l1 = thecontrols.createLeftLabel("Zoom In / Out");
+        HBox r1 = thecontrols.createCustomText("Mouse Wheel");
+
+        VBox l2 = thecontrols.createLeftLabel("");
+        HBox r2 = thecontrols.createCustomText("Shift");
+        r2.getChildren().addAll(new ImageView(myPlusDir), thecontrols.customIco(AwesomeIconSolid.ARROW_UP),
+                thecontrols.customIco(AwesomeIconSolid.ARROW_DOWN));
+
+        VBox l3 = thecontrols.createLeftLabel("Zoom Fast");
+        HBox r3 = thecontrols.createCustomText("");
+        r3.getChildren().addAll(thecontrols.customImageView(mouseMiddle), new ImageView(myPlusDir),
+                thecontrols.customIco(AwesomeIconSolid.ARROWS_ALT_V));
+
+        VBox l4 = thecontrols.createLeftLabel("Zoom Way In / Out");
+        HBox r4 = thecontrols.createCustomText("");
+        r4.getChildren().addAll(thecontrols.customImageView(mouseLeft), new ImageView(myPlusDir),
+                thecontrols.customImageView(mouseRight), new ImageView(myPlusDir),
+                thecontrols.customIco(AwesomeIconSolid.ARROWS_ALT_V));
+
+        VBox l5 = thecontrols.createLeftLabel("Draw Zoom Box");
+        HBox r5 = thecontrols.createCustomText("Ctrl");
+        r5.getChildren().addAll(new ImageView(myPlusDir), thecontrols.customImageView(mouseLeft));
+
+        dispArea.add(l1, 0, 1);
+        dispArea.add(r1, 2, 1);
+        dispArea.add(l2, 0, 2);
+        dispArea.add(r2, 2, 2);
+        dispArea.add(l3, 0, 3);
+        dispArea.add(r3, 2, 3);
+        dispArea.add(l4, 0, 4);
+        dispArea.add(r4, 2, 4);
+        dispArea.add(l5, 0, 5);
+        dispArea.add(r5, 2, 5);
+        return mainBox;
+    }
+
+    /**
+     * Creates a sub-window containing Map Movement Controls.
+     * 
+     * @return mainBox the sub-window contained in a GridPane.
```

Comment:
No need to give the name of the return variable

Reply:
updated